package.json

```diff
@@ -36,7 +36,7 @@
     "lbh-frontend": "^3.5.3",
     "lodash.debounce": "^4.0.8",
     "lodash.get": "^4.4.2",
-    "next": "^11.1.1",
+    "next": "^12.0.8",
```

Comment:
Just a little note, have you gone over the upgrade guide for 11 to 12 to make sure everything will work? https://nextjs.org/docs/upgrading#upgrading-from-11-to-12

Reply:
Thanks Lawrence, I hadn't spotted that documentation
I've dropped the next version to 11.1.3 and created ticket SCT-1666 to handle the upgrade to version 12